luigi/contrib/postgres.py

```diff
@@ -207,24 +207,21 @@ def create_marker_table(self):
         connection.autocommit = True
         cursor = connection.cursor()
         if self.use_db_timestamps:
-            sql = """ CREATE TABLE {marker_table} (
+            sql = """ CREATE TABLE IF NOT EXISTS {marker_table} (
                       update_id TEXT PRIMARY KEY,
                       target_table TEXT,
                       inserted TIMESTAMP DEFAULT NOW())
-                """.format(marker_table=self.marker_table)
+                  """.format(marker_table=self.marker_table)
         else:
-            sql = """ CREATE TABLE {marker_table} (
+            sql = """ CREATE TABLE IF NOT EXISTS {marker_table} (
                       update_id TEXT PRIMARY KEY,
                       target_table TEXT,
                       inserted TIMESTAMP);
                   """.format(marker_table=self.marker_table)
         try:
             cursor.execute(sql)
         except psycopg2.ProgrammingError as e:
```

Comment:
Flake8 doesn't like the assignment, but non-use of `e` here.

I think we can just remove the try/except here for `cursor.execute(sql)`.